ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.m

```diff
@@ -159,14 +159,26 @@ - (UIView *)miniAppViewWithName:(NSString *)name properties:(NSDictionary *)prop
     return rootView;
 }
 
-- (UIView *)miniAppViewWithName:(NSString *)name properties:(NSDictionary *_Nullable)properties
-            sizeFlexibility:(NSInteger)sizeFlexibilty {
+- (UIView *)miniAppViewWithName:(NSString *)name
+                     properties:(NSDictionary *_Nullable)properties
+                sizeFlexibility:(NSInteger)sizeFlexibilty {
     RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:self.bridge moduleName:name initialProperties:properties];
     rootView.sizeFlexibility = (RCTRootViewSizeFlexibility)sizeFlexibilty;
     rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
     return rootView;
 }
 
+- (UIView *)miniAppViewWithName:(NSString *)name
+                     properties:(NSDictionary *_Nullable)properties
+                sizeFlexibility:(NSInteger)sizeFlexibilty
+                       delegate:(id<MiniAppViewDelegate> _Nullable)delegate {
+    RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:self.bridge moduleName:name initialProperties:properties];
+    rootView.sizeFlexibility = (RCTRootViewSizeFlexibility)sizeFlexibilty;
+    rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
+    rootView.delegate = delegate;
```

Comment:
Don't we need to change the order like below to get the `rootViewDidChangeIntrinsicSize` to get triggered when the `sizeFlexibility` is set?
```
rootView.delegate = delegate;
rootView.sizeFlexibility = (RCTRootViewSizeFlexibility)sizeFlexibilty;
    rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
```

Reply:
Good point. In this case we don't - setting `sizeFlexibility` simply sets the property and calls `setNeedsLayout`, which only invalidate current layout but does not force an immediate update, instead waits for the next update cycle. The `rootViewDidChangeIntrinsicSize` delegate method, as a result, won't be called until the next update cycle. Therefore, the order of setting `sizeFlexibility ` and `delegate` does not matter.

Reply:
Thanks for the clarification.